### ISSUE TITLE
Include Domain name in Directory field

### ISSUE THREAD

**Issue by balajibr** [projectkudu/slingshot#40]
If you have multiple directories in your subscription, it is not possible to choose the right one as the drop down lists "Default Directory" for each one. We might need to append Domain name as well within brackets for each directory listing.
![azuredeploy](https://cloud.githubusercontent.com/assets/9376444/7021594/edbfc16e-dd40-11e4-8df1-24b14aa9afff.PNG)

**Comment by davidebbo** [comment]
Yes, we should do the same as in https://resources.azure.com. @balajibr can you check that https://resources.azure.com/ works correctly with that same account?

**Comment by balajibr** [comment]
Yes David - ARM explorer does show the domain name in brackets.
